openhcl/underhill_attestation/src/secure_key_release.rs

```diff
@@ -154,15 +154,18 @@ pub async fn request_vmgs_encryption_keys(
             Ok(WrappedKeyVmgsEncryptionKeys {
                 rsa_aes_wrapped_key: _,
                 wrapped_des_key: _,
-            }) if i == (max_retry - 1) => break,
+            }) if i == (max_retry - 1) => {
+                tracing::error!("VMGS key-encryption failed after max number of attempts");
+                break;
+            }
             Ok(WrappedKeyVmgsEncryptionKeys {
                 rsa_aes_wrapped_key: _,
                 wrapped_des_key: _,
             }) => {
                 tracing::warn!(
                     CVM_ALLOWED,
                     retry = i,
-                    "VMGS key-encryption key is not released"
+                    "Attempt to get VMGS key-encryption failed"
```

Comment:
nit
```suggestion
                    "Attempt to get VMGS key-encryption but failed"
```

Reply:
Updated to "Failed to get VMGS key-encryption"